### PR TITLE
[14.0] [IMP] pms manage pricelist board services

### PR DESCRIPTION
**Pull request by miguelpadin**

### CODE REVIEW

File: pms/models/product_pricelist_item.py [hunk 1]
```diff
@@ -18,26 +18,18 @@ class ProductPricelistItem(models.Model):
...
-        help="Those included in Board Services",
-    )
-    board_service_room_type_ids = fields.Many2many(
-        string="Board Services",
-        help="""Specify a Board services on Room Types.""",
+    board_service_room_type_id = fields.Many2one(
+        string="Board Service",
+        help="Specify a Board services on Room Types.",
         comodel_name="pms.board.service.room.type",
-        relation="board_service_pricelist_item_rel",
```
Comment (DarioLodeiros) [inline pms/models/product_pricelist_item.py:37]: Why remove relation, column1 and column2 attributes?

**Comment by miguelpadin** [comment]
https://user-images.githubusercontent.com/6873385/120773695-def31f80-c521-11eb-8e2d-cdcdb095ad19.mp4

**Comment by miguelpadin** [comment]
https://user-images.githubusercontent.com/6873385/120779285-33e56480-c527-11eb-95ff-fa45400abfc0.mp4

**Review by DarioLodeiros — APPROVED** [review]

**Pull request merged**